pkg/cmd/registry/list/list.go

```diff
@@ -64,6 +64,15 @@ func NewListCommand(f *factory.Factory) *cobra.Command {
 			if opts.outputFormat != "" && !flagutil.IsValidInput(opts.outputFormat, flagutil.ValidOutputFormats...) {
 				return flag.InvalidValueError("output", opts.outputFormat, flagutil.ValidOutputFormats...)
 			}
+			if opts.page < 1 {
+				return opts.localizer.MustLocalizeError("registry.list.flag.page",
```

Comment:
Use of `if` looks great, however the error message returned should be changed. Current message doesn't tell about the error, rather simply displays the flag description.

![Screenshot from 2021-10-01 16-29-00](https://user-images.githubusercontent.com/23582438/135609370-929d4885-1e72-4a3b-95aa-a39589a78f80.png)

Reply:
@rkpattnaik780 I don't understand how the rendering is happening for `opts.localizer.MustLocalizeError`. I want to know how its parameters work. Could you guide me in the same?

Reply:
@amoghrajesh We will take care of that. @rkpattnaik780 will take your PR and do all required changes.

Reply:
@wtrocki got it. Could you merge this or label it so it can be counted as a Hactoberfest PR for me?

Reply:
Yes. It will be merged today! I promise. Thank you so much for your contribution!

Reply:
Hi @amoghrajesh you can refer to this [commit](https://github.com/redhat-developer/app-services-cli/pull/1163/commits/3d2ca33976195e85f2f4f3502adea70b06fd5152) to understand more about how this project implements localization. Basically we keep all the localized strings [here](https://github.com/redhat-developer/app-services-cli/tree/main/pkg/localize/locales). 
Hope this helps :) 
Thanks!